apps/go/requester/config.sample.json

```diff
@@ -7,16 +7,21 @@
     "urls": [
       "http://localhost:8081"
     ],
-    "retries": 3,
+    "retries": 1,
     "min_backoff": 10,
     "max_backoff": 60,
-    "req_per_sec": 10
+    "req_per_sec": 10,
+    "session_tolerance": 1
   },
   "log_level": "debug",
   "temporal": {
     "host": "localhost",
     "port": 7233,
     "namespace": "pocket-ml-testbench",
-    "task_queue": "requester-local"
+    "task_queue": "requester"
```

Comment:
if "evaluator" has its own section, shouldnt "requester" have also its own?

Reply:
i will evaluator key inside of temporal